master/internal/trials/postgres_trials.go

```diff
@@ -70,8 +71,8 @@ func MetricsTimeSeries(trialID int32, startTime time.Time,
 		case db.MetricTypeBool:
 			cast = "boolean"
 		}
-		subq = subq.ColumnExpr("(metrics->?->>?)::? as ?",
-			metricsObjectName, metricName, bun.Safe(cast), bun.Ident(metricName))
+		subq = subq.ColumnExpr("(metrics->?->>?)::? as ?", metricsObjectName,
+			metricName, bun.Safe(cast), bun.Ident(strings.ReplaceAll(metricName, ".", "·")))
```

Comment:
blocking: ```db.ApplyPolymorphicFilter``` needs to be updated to have this escaping too right?

Reply:
✅ agreed